pkg/logger/logger.go

```diff
@@ -88,7 +88,7 @@ const (
 // Tracee offers aggregation and filtering support on top of any logger implementation complying to it's interface.
 type LoggingConfig struct {
 	Logger        LoggerInterface
-	loggerConfig  LoggerConfig
+	LoggerConfig  LoggerConfig
```

Comment:
I don't like this refactoring, but it had to be done because [flags create a LoggingConfig](https://github.com/aquasecurity/tracee/blob/main/pkg/cmd/flags/logger.go#L292), and if this field wasn't exposed it would be nil, generating a panic when you change the log level. But it is weird to allow the user to create a struct but have methods depending on it, we should enforce it by perhaps having these fields private and a constructor function. Nonetheless, I'll create a separate issue to handle this refactor.